docs/appendix/local-configuration-file.md

```diff
@@ -1,230 +1,224 @@
+
+- Linux: <file>/etc/viam.json</file>
+- macOS: <file>/opt/homebrew/etc/viam.json</file>
+
+You can also use the [example configuration file](#example-json-configuration-file) below to base your robot's configuration on.
+
+{{% alert title="Note" color="note" %}}
+This process is not required if your robot is connected to the Viam app.
+When connected, any configuration changes you make in the app are propagated to your robot automatically.
+If your robot temporarily disconnects from the internet, its configuration is cached locally, and any configuration changes you may have made in the app are propagated to your robot once it reconnects.
+{{% /alert %}}
+
+## Example JSON configuration file
+
+If you want to create your own JSON configuration file without using [the Viam app](https://app.viam.com), you can start with the following example file.
+This file contains some basic example [component](/components/) and [service](/services/) configurations, as well as an example of a {{< glossary_tooltip term_id="process" text="process" >}}:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "network": {
+    "fqdn": "something-unique",
+    "bind_address": ":8080"
+  },
+  "components": [
+    {
+      "name": "arm1",
+      "type": "arm",
+      "model": "fake",
+      "attributes": {
+        "arm-model": "xArm6"
+      }
+    },
+    {
+      "name": "audio_input1",
+      "type": "audio_input",
+      "model": "fake"
+    },
+    {
+      "name": "base1",
+      "type": "base",
+      "model": "fake"
+    },
+    {
+      "name": "board1",
+      "type": "board",
+      "model": "fake",
+      "attributes": {
+        "analogs": [
+          {
+            "name": "analog1",
+            "pin": "0"
+          }
+        ],
+        "digital_interrupts": [
+          {
+            "name": "di1",
+            "pin": "14"
+          }
+        ]
+      }
+    },
+    {
+      "name": "camera1",
+      "type": "camera",
+      "model": "fake"
+    },
+    {
+      "name": "encoder1",
+      "type": "encoder",
+      "model": "fake",
+      "attributes": {
+        "update_rate_msec": 200
+      }
+    },
+    {
+      "name": "gantry1",
+      "type": "gantry",
+      "model": "fake"
+    },
+    {
+      "name": "generic1",
+      "type": "generic",
+      "model": "fake"
+    },
+    {
+      "name": "gripper1",
+      "type": "gripper",
+      "model": "fake"
+    },
+    {
+      "name": "input_controller1",
+      "type": "input_controller",
+      "model": "fake"
+    },
+    {
+      "name": "motor1",
+      "type": "motor",
+      "model": "fake",
+      "attributes": {
+        "encoder": "encoder1",
+        "pins": {
+          "a": "1",
+          "b": "2",
+          "pwm": "3"
+        },
+        "ticks_per_rotation": 100
+      },
+      "depends_on": ["board1", "encoder1"]
```

Comment:
This is odd. Does it have to do that?

Reply:
Yes it does that to conserve space if the array is short enough to fit on one line

Reply:
My concern is just that this is different from what the app does--I wish we could get it to match the app so that people can seamlessly cross-reference

Reply:
But I suppose not a common issue since implicit dependencies are the norm, and most places where we still have anything in depends_on could be removed

Reply:
if there were more here it would split onto multiple lines, so we could do that or change the names if we want to ensure it's on two lines, but let's do that separately.